src/v3/src/transformer/idxTransformerMapping.ts

```diff
@@ -194,6 +195,14 @@ const TransformerMap: {
         showForgotPassword: true,
       },
     },
+    [AUTHENTICATOR_KEY.PASSWORD]: {
+      transform: transformIdentify,
+      buttonConfig: {
+        showDefaultSubmit: false,
+        showDefaultCancel: false,
+        showForgotPassword: true,
+      },
+    },
```

Comment:
This is not required, can be removed. See: https://github.com/okta/okta-signin-widget/blob/master/src/v2/view-builder/ViewFactory.ts#L117-L119